sql-cloudant/README.md

```diff
@@ -1,24 +1,10 @@
-Spark Cloudant Connector
-================
+A library for reading data from Cloudant databases using Spark SQL and Spark Streaming. 
 
-Cloudant integration with Spark as Spark SQL external datasource, and Spark Streaming as a custom receiver. 
-
-
-##  Contents:
-0. [Linking](#Linking)
-1. [Implementation of RelationProvider](#implementation-of-relationProvider)
-2. [Implementation of Receiver](#implementation-of-Receiver)
-3. [Sample applications](#Sample-application)
-    1. [Using SQL In Python](#Using-SQL-In-Python)
-    2. [Using SQL In Scala](#Using-SQL-In-Scala)
-    3. [Using DataFrame In Python](#Using-DataFrame-In-Python)
-    4. [Using DataFrame In Scala](#Using-DataFrame-In-Scala)
-    5. [Using Streams In Scala](#Using-Streams-In-Scala)
-4. [Configuration Overview](#Configuration-Overview)
-5. [Known limitations and areas for improvement](#Known-limitations)
-
-
-<div id='Linking'/>
+[IBM® Cloudant®](https://cloudant.com) is a document-oriented DataBase as a Service (DBaaS). It stores data as documents 
```

Comment:
Can we make sure we also mention that this works for vanilla Apache CounchDB as well.